packages/contracts/package.json

```diff
@@ -8,6 +8,7 @@
     "dist/types-ovm/*.ts",
     "artifacts/**/*.json",
     "artifacts-ovm/**/*.json",
+    "deployments/**/*.json",
```

Comment:
Why are you adding this?

Reply:
this contains all the L1 artifacts & addresses so it needs to be in the published package